package.json

```diff
@@ -2,48 +2,55 @@
   "name": "sauce-testcafe-runner",
   "version": "0.0.0",
   "description": "Sauce Labs Testcafe runner",
-  "author": "<dev@saucelabs.com>",
+  "author": "<devx@saucelabs.com>",
   "license": "MIT",
   "private": true,
   "bin": "./bin/testcafe",
-  "main": "src/console-wrapper.js",
+  "main": "lib/console-wrapper.js",
   "scripts": {
-    "test": "node src/console-wrapper.js",
-    "lint": "eslint tests/ src/",
+    "build": "tsc",
+    "test": "tsc; node lib/console-wrapper.js",
+    "lint": "eslint tests/ src/**/*.ts",
     "unit-test": "jest --env node"
   },
   "keywords": [
     "testcafe"
   ],
   "dependencies": {
     "@saucelabs/testcomposer": "^1.0.0",
+    "@tsconfig/node18": "^1.0.1",
     "dotenv": "16.0.3",
     "mocha-junit-reporter": "^2.2.0",
     "sauce-testrunner-utils": "1.0.0",
     "testcafe": "2.3.1",
     "testcafe-browser-provider-ios": "0.5.0",
     "testcafe-reporter-saucelabs": "1.0.1",
-    "typescript": "4.9.5",
+    "typescript": "^5.0.4",
+    "typescritp": "^1.0.0",
```

Comment:
That seems like a typo and actually a spam library of sorts.

Reply:
ohh..